tutorials/RecSys22tutorial/03_Customize_Merlin_Models.ipynb

```diff
@@ -36,7 +36,7 @@
    "id": "ebc9c8c6-9fab-435f-958b-923b0f3d6ff4",
```

Comment:
<p>I would print embeddings(batch) once? how many categorical features do we have?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/publications/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>yes we can do that it is just a very long output.</p>